pkg/limayaml/defaults.go

```diff
@@ -955,14 +955,21 @@ func executeHostTemplate(format, instDir string, param map[string]string) (bytes
 	tmpl, err := template.New("").Parse(format)
 	if err == nil {
 		limaHome, _ := dirnames.LimaDir()
+		globalTempDir := "/tmp"
+		if runtime.GOOS == "windows" {
+			// There is no global temp dir on Windows, and the system temp dir is not writable by the user.
```

Comment:
"global temp dir" != "system temp dir" ?

```suggestion
			// On Windows, the global temp dir (C:\TEMP) is not writable by the user.
```

Reply:
Maybe `C:\TEMP` → `C:\Windows\Temp`, idk

Reply:
`C:\TEMP` is not a thing on Windows.

I was talking about `%SystemRoot%\SystemTemp`.

There is normally also a `%SystemRoot%\Temp` directory, but that may also not be writable by regular users. It does seem to be writable by local administrators, even when not running elevated.

I'll update the comment, but will use `C:\Windows\Temp` as the directory name.